Gemfile.lock

```diff
@@ -254,15 +210,13 @@ GEM
       childprocess (~> 0.5)
       rubyzip (~> 1.0)
       websocket (~> 1.0)
-    simplecov (0.16.1)
-      docile (~> 1.1)
+    simplecov (0.13.0)
```

Comment:
This downgrade of `simplecov` is strange to me. Was this caused by something that I can't see at the moment?

Reply:
Looks like the codeclimate-test-reporter 1.0.8 has a dependency on simplecov at <=0.13.0:

[rubygems page](https://rubygems.org/gems/codeclimate-test-reporter/versions/1.0.8)

Not sure why they put that restriction on it for 1.0.8, since it wasn't there for 1.0.7.. 
Either way, codeclimate-test-reporter is deprecated, so shouldn't spend too many brain cells on this one.  I'll move codeclimate back to 1.0.7 and then move simplecove back to  0.16.1, So it shouldn't have any diff anymore.